addons/test_mail/tests/test_mail_push.py

```diff
@@ -81,8 +81,8 @@ def test_notify_by_push(self, push_to_end_point):
 
     @patch.object(odoo.addons.mail.models.mail_thread, 'push_to_end_point')
     def test_notify_by_push_channel(self, push_to_end_point):
-        """ Test various use case with discuss.channel. Chat and group channels
-        sends push notifications, channel not. """
+        """ Test various use case with discuss.channel. Channel, chat and group channels
+        not sends push notifications """
```

Comment:
I don't think changes related to push notifications are correct. We should keep and extend them instead.